File: scala/scala-impl/testdata/refactoring/rename3/settersWithoutVar/before/tests/Foo.java
package tests;

public class Foo {
    public static void main(String[] args) {
        int b1 = Bar.bar();
        int b2 = Bar$.MODULE$.bar();
        Bar$.MODULE$.bar_/*caret*/$eq(3);
        Bar2.bar_$/*caret*/eq(2);
        Bar2.bar();
        Bar2$.MODULE$.bar();
        Bar2$.MODULE$.bar_/*caret*/$eq(4);
    }
}

File: scala/scala-impl/testdata/refactoring/rename3/classAuxConstructorEndMarker/before/tests/Foo.scala
package tests

class Foo(i: Int):
  def bar(x: Int) = i + x
  def this/*caret*/() =
    this(42)
  end this
  def /*caret*/this(str: String) =
    this(str.length)
  end this
end Foo

object Test:
  val foo1 = Foo(1)
  val foo2 = Foo()
  val foo3 = Foo("foo")
end Test
